packages/m/msgpack-cxx/xmake.lua

```diff
@@ -24,21 +22,7 @@ package("msgpack-cxx")
     end)
 
     on_install(function (package)
-        local configs = {"-DMSGPACK_BUILD_EXAMPLES=OFF", "-DMSGPACK_BUILD_TESTS=OFF", "-DMSGPACK_BUILD_DOCS=OFF", "-DMSGPACK_USE_STATIC_BOOST=ON"}
-        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
-        table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
-        if package:config("std") ~= "cxx98" then
-            table.insert(configs, "-DMSGPACK_" .. package:config("std"):upper() .. "=ON")
-        end
-        if package:config("boost") then
-            table.insert(configs, "-DMSGPACK_USE_BOOST=ON")
-            if is_plat("windows") then
-                table.insert(configs, "-DBoost_USE_STATIC_RUNTIME=" .. (package:config("vs_runtime"):startswith("MT") and "ON" or "OFF"))
-            end
-        else
-            table.insert(configs, "-DMSGPACK_USE_BOOST=OFF")
-        end
-        import("package.tools.cmake").install(package, configs)
+        os.cp("include", package:installdir())
```

Comment:
Whenever possible, use cmake even if it is headeronly, as `cmake install` may also install an additional `xxx.cmake` import file that will allow other cmake packages to find it better.

Reply:
I get `package(msgpack-cxx): links not found!` errors when I use `cmake install` for the checks

Reply:
you need only add `set_kind("library", {headeronly = true})`